File: SISMUEBLUM JAVA/proveedor.java
public class proveedor {
	private int idProveedor;
	private String nombre;
	private String direccion;
	private productos prod;
	private String pais;
	private String telefono
	
	public proveedor(){
		
	}
	
	public proveedor(int idEnt, String nomEnt, productos p, String paisEnt, String tlfEnt){
		this.idProveedor = idEnt;
		this.nombre = nomEnt;
		this.prod = p;
		this.pais = paisEnt;
		this.telefono = tlfEnt;
	}
	
	public int getIdProveedor(){
		return idProovedor;
	}
	
	public void setIdProveedor(int idEnt){
		this.idProovedor = idEnt;
	}
	
	public String getNombre(){
		return nombre;
	}
	
	public void setNombre(String nomEnt){
		this.nombre = nomEnt;
	}

	public String getDireccion(){
		return direccion;
	}
	
	public void setDireccion(String dirEnt){
		this.direccion = dirEnt;
	}
	
	public String getPais(){
		return pais;
	}
	
	public void setPais(String paisEnt){
		this.pais = paisEnt;
	}
	
	public String getTelefono(){
		return telefono;
	}
	
	public void setTelefono(String tlfEnt){
		this.telefono = tlfEnt;
	}	
}

File: SISMUEBLUM C#/Proveedor.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
public class Proveedor {
	private int idProveedor;
	private string nombre;
	private string direccion; //metodo
	private List<Producto> listProd;
	private string pais;
	private string telefono;


	public Proveedor(){
		listProd = new List<Producto>();
	}
	
	public Proveedor(int idEnt, string nomEnt, string direccion, string paisEnt, string tlfEnt){
		listProd = new List<Producto>();
		this.idProveedor = idEnt;
		this.direccion = direccion;
		this.nombre = nomEnt;
		this.pais = paisEnt;
		this.telefono = tlfEnt;
	}
	
	~Proveedor(){
	}
	
	public int IdProveedor{
		get{
			return idProveedor;
		}
		set{
			idProveedor = value;
		}
	}
	
	
	public string Nombre{
		get{
			return nombre;
		}
		set{
			nombre = value;
		}
	}

	public string Direccion{
		get{
			return direccion;
		}
		set{
			direccion = value;
		}
	}	
	
	public string Pais{
		get{
			return pais;
		}
		set{
			pais = value;
		}
	}
		
	public string Telefono{
		get{
			return telefono;
		}
		set{
			telefono = value;
		}
	}
	
	public void registrarProducto(){
		
	}
	
	public void eliminarProducto(){		
		
	}
}
